dart/lib/src/enricher/_io_enricher.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:async';
+import 'dart:io';
+
+import '../protocol.dart';
+
+import 'enricher.dart';
+
+final Enricher instance = IoEnricher();
+
+/// Enriches [SentryEvents] with various kinds of information.
+/// Uses Darts [Platform](https://api.dart.dev/stable/dart-io/Platform-class.html)
+/// class to read information.
+class IoEnricher implements Enricher {
+  @override
+  FutureOr<SentryEvent> apply(SentryEvent event) {
+    final contexts = event.contexts.copyWith(
+      operatingSystem: _getOperatingSystem(event.contexts.operatingSystem),
+      runtimes: _getRuntimes(event.contexts.runtimes),
+    );
+
+    return event.copyWith(
+      contexts: contexts,
+      extra: _getExtras(event.extra),
+    );
+  }
+
+  List<SentryRuntime> _getRuntimes(List<SentryRuntime>? runtimes) {
+    final dartRuntime = SentryRuntime(name: 'Dart', version: Platform.version);
+    final flutterRuntime = SentryRuntime(name: 'Flutter');
```

Comment:
Not Flutter

Reply:
This is the Dart SDK, not the Flutter SDK. So including it here would be wrong. It should go into the FlutterEnricher. If at all, because, as you said, there's no Flutter version info.